style.css

```diff
@@ -319,10 +337,23 @@ nav {
   background: var(--button-pressed-color);
 }
 
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100vh;
+  margin-bottom: 59px;
+  background: url('./assets/contact_bg.png') no-repeat;
+  background-position: 100% 80%, left bottom;
+}
+
```

Comment:
[OPTIONAL] Well done overall 💯 but in order to improve the design, I'd suggest adding some padding to the inputs, this is what you have for now 👇🏽 

![Screenshot 2023-05-04 at 03 04 04](https://user-images.githubusercontent.com/45739963/236133560-55b16d06-3bc6-448f-b0e4-10d6b6ff6a47.png)

Reply:
Alright, thank you.